### ISSUE TITLE
New lines keep being added within script block

### ISSUE THREAD

**Issue by bberg11** [Shopify/theme-tools#162]
**Describe the bug**
`<script>` tag in liquid is adding additional new lines after opening tag and before closing tag on each run of the formatter. In other words, if the formatter runs on the file 5 times, there will be 5 blank lines before and after the contents of the script tag.

**Unformatted source**
```liquid
<script>
  if (condition) {
    ...
  }
</script>
```

**Expected output**
```liquid
<script>
  if (condition) {
    ...
  }
</script>
```

**Actual output**
```liquid
<script>

  if (condition) {
    ...
  }

</script>
# then on the next format run
<script>


  if (condition) {
    ...
  }


</script>
# and on and on
```

**Debugging information**
 - MacOS 13.4.1
 - "@shopify/prettier-plugin-liquid": "^1.0.6"

**Additional context**
I've only seen this on this script tag. Perhaps the condition block being the only JS within it is the issue?

**Comment by charlespwd** [comment]
Ah nevermind found a case that reproduces! It's Liquid outside of strings + backticks it seems.

**Comment by charlespwd** [comment]
Fixed in`@shopify/prettier-plugin-liquid@1.3.2` and the Shopify Liquid VS Code extension `1.11.11`
